modules/auxiliary/scanner/rdp/cve_2019_0708.rb

```diff
@@ -0,0 +1,545 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'digest'
+require 'rc4'
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::Tcp
+  include Msf::Auxiliary::Scanner
+  include Msf::Auxiliary::Report
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'CVE-2019-0708 Microsoft Remote Desktop RCE Checker',
+      'Description'    => %q{
+        This module checks a range of hosts for the CVE-2019-0708 vulnerability.
+        This should not cause a DoS on the target.
```

Comment:
Should probably have some vulnerability info

Reply:
yes please. Also, I would hope it wouldn't cause a DoS (this should go without saying!)

Reply:
Might be worth adding [module traits](https://github.com/rapid7/metasploit-framework/wiki/Definition-of-Module-Reliability,-Side-Effects,-and-Stability).

Reply:
Added more detailed description and module traits.